src/tagmanager/tm_workspace.c

```diff
@@ -453,19 +419,19 @@ static GList *lookup_includes(const gchar **includes, gint includes_count)
 #ifdef HAVE_GLOB_H
 	globbuf.gl_offs = 0;
 
-	if (includes[0][0] == '"')	/* leading \" char for glob matching */
+	if (sources[0][0] == '"')	/* leading \" char for glob matching */
 	{
-		for (i = 0; i < includes_count; i++)
+		for (i = 0; i < sources_count; i++)
 		{
-			size_t dirty_len = strlen(includes[i]);
+			size_t dirty_len = strlen(sources[i]);
 			gchar *clean_path;
 
 			if (dirty_len < 2)
 				continue;
 
 			clean_path = g_malloc(dirty_len - 1);
 
-			strncpy(clean_path, includes[i] + 1, dirty_len - 1);
+			strncpy(clean_path, sources[i] + 1, dirty_len - 1);
```

Comment:
Skips first character of source, see above.

Reply:
Old code so I would ignore this problem for now.

But IIUC this is only a problem for the second and later arguments as the first should be enclosed by `"` (so `clean_path` strips `"`), right?

Plus, as @techee mentions, the condition is probably never true. Best to leave globbing to the shell really (or clever find+xargs tricks)